stack/adding-a-new-package.rst

```diff
@@ -2,8 +2,8 @@
 Adding a New Package to the Build
 #################################
 
-New packages intended for distribution to end users should generally be added as a dependency of a "top-level product:" these are the roots of the LSST package hierarchy.
-They include ``lsst_apps``, ``lsst_distrib``, ``qserv_distrib`` and ``lsst_sims``.
+New packages intended for distribution to end users should generally be added as a dependency of a "top-level product": these are the roots of the LSST package hierarchy.
+They include ``lsst_apps`` (for packages that are used by production pipelines), ``lsst_distrib`` (for packages that are distributed alongside the pipelines but not used directly by them), ``qserv_distrib`` and ``lsst_sims``.
```

Comment:
Aren't `jointcal` and `fgcmcal` only specified in lsst_distrib? That makes the `lsst_apps` definition a bit inconsistent doesn't it?

Reply:
Since this has never been fully resolved (see RFC-305, withdrawn), I left it out.